### ISSUE TITLE
Demo steps lead to stack trace - undefined method vpc_id

### ISSUE THREAD

**Issue by et304383** [tongueroo/ufo#48]
Following the demo here: http://ufoships.com/docs/tutorial-ufo-init/

I get this on the first ufo init command:

```
$ ufo init --app=demo --image=tongueroo/demo-ufo
Traceback (most recent call last):
        20: from /usr/local/bin/ufo:23:in `<main>'
        19: from /usr/local/bin/ufo:23:in `load'
        18: from /var/lib/gems/2.5.0/gems/ufo-4.1.4/exe/ufo:14:in `<top (required)>'
        17: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor/base.rb:466:in `start'
        16: from /var/lib/gems/2.5.0/gems/ufo-4.1.4/lib/ufo/command.rb:43:in `dispatch'
        15: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
        14: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
        13: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
        12: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor.rb:40:in `block in register'
        11: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:115:in `invoke'
        10: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor/group.rb:232:in `dispatch'
         9: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `invoke_all'
         8: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `map'
         7: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `each'
         6: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `block in invoke_all'
         5: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
         4: from /var/lib/gems/2.5.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
         3: from /var/lib/gems/2.5.0/gems/ufo-4.1.4/lib/ufo/init.rb:52:in `set_network_options'
         2: from /var/lib/gems/2.5.0/gems/ufo-4.1.4/lib/ufo/network/helper.rb:10:in `configure_network_settings'
         1: from /var/lib/gems/2.5.0/gems/memoist-0.16.0/lib/memoist.rb:170:in `vpc_id'
/var/lib/gems/2.5.0/gems/ufo-4.1.4/lib/ufo/network/fetch.rb:18:in `vpc_id': undefined method `vpc_id' for nil:NilClass (NoMethodError)
```

**Comment by tongueroo** [comment]
@et304383 I see, this is because there is no default vpc in your particular setup. This makes sense, you might want to delete the default vpc as a security measure. 

Fixed in #49 and released in v4.1.5 by providing a instructional message to use the `--vpc-id` option. More info here in the ufo init CLI reference doc: http://ufoships.com/reference/ufo-init/

* [CHANGELOG](https://github.com/tongueroo/ufo/blob/master/CHANGELOG.md)

**Comment by et304383** [comment]
Thanks for the quick turn-around!
